src/Microsoft.AspNetCore.WebHooks.Receivers/Filters/WebHookGetResponseFilter.cs

```diff
@@ -142,7 +141,7 @@ public void OnResourceExecuted(ResourceExecutedContext context)
                     CultureInfo.CurrentCulture,
                     Resources.General_MissingQueryParameter,
                     getMetadata.ChallengeQueryParameterName);
-                var noChallenge = WebHookResultUtilities.CreateErrorResult(message);
+                var noChallenge = new BadRequestObjectResult(message);
```

Comment:
This overlaps #205 because I'd already fixed up the renamed files. Doesn't really matter which goes in first&hellip;